ui/components/src/LineChart/LineChart.tsx

```diff
@@ -95,7 +95,7 @@ export function LineChart({
   const chartsTheme = useChartsTheme();
   const chartRef = useRef<EChartsInstance>();
   const [showTooltip, setShowTooltip] = useState<boolean>(true);
-  const [pinTooltip, setPinTooltip] = useState<boolean>(false);
+  const [tooltipPinned, setTooltipPinned] = useState<boolean>(false);
```

Comment:
Could do `isTooltipPinned` if you like having the `is` / some verb in front of booleans (I forget it we have style guide around that, I think I had that at past company)